test/integration/restart_task_test.rb

```diff
@@ -190,6 +190,32 @@ def test_restart_failure
       in_order: true)
   end
 
+  def test_restart_failure_can_be_ignored
+    success = deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "web.yml.erb"]) do |fixtures|
```

Comment:
This is my ultimate nitpick: `success` is what we like, but so far we only know it's a `result`